composer.json

```diff
@@ -58,18 +58,18 @@
         "symfony-cmf/media-bundle":             "~1.2",
         "symfony-cmf/menu-bundle":              "~1.2",
         "symfony-cmf/routing-bundle":           "~1.2",
-        "symfony/assetic-bundle":               "~2.3",
-        "symfony/event-dispatcher":             "~2.3",
-        "symfony/expression-language":          "~2.4",
-        "symfony/form":                         "~2.3",
-        "symfony/framework-bundle":             "~2.3",
-        "symfony/intl":                         "~2.3",
-        "symfony/monolog-bundle":               "~2.3",
-        "symfony/options-resolver":             "~2.3",
-        "symfony/property-access":              "~2.3",
+        "symfony/assetic-bundle":               "~2.6",
+        "symfony/event-dispatcher":             "~2.7",
+        "symfony/expression-language":          "~2.7",
+        "symfony/form":                         "~2.7",
+        "symfony/framework-bundle":             "~2.7",
+        "symfony/intl":                         "~2.7",
+        "symfony/monolog-bundle":               "~2.7",
+        "symfony/options-resolver":             "~2.7",
+        "symfony/property-access":              "~2.7",
         "symfony/swiftmailer-bundle":           "~2.3",
```

Comment:
It's just a temporary solution, this PR should not make the bump to PHP5.4 / Sf2.7 alone, it should be made in another PR and then rebased (after removing 6aafbb1592f83730521412843323872d0b100cfb).